pkg/securitypolicy/securitypolicy.go

```diff
@@ -127,3 +81,94 @@ func NewSecurityPolicyState(base64Policy string) (*SecurityPolicyState, error) {
 		EncodedSecurityPolicy: encodedSecurityPolicy,
 	}, nil
 }
+
+type SecurityPolicy struct {
+	// Flag that when set to true allows for all checks to pass. Currently used
+	// to run with security policy enforcement "running dark"; checks can be in
+	// place but the default policy that is created on startup has AllowAll set
+	// to true, thus making policy enforcement effectively "off" from a logical
+	// standpoint. Policy enforcement isn't actually off as the policy is "allow
+	// everything:.
+	AllowAll bool `json:"allow_all"`
+	// One or more containers that are allowed to run
+	Containers Containers `json:"containers"`
+}
+
+type Containers struct {
+	Length   int                  `json:"length"`
```

Comment:
do we still need this even though we're using a custom `MarshalJSON`?

Reply:
Yes

Reply:
I see that the `MarshalJSON` code is using an inline struct that overrides the value of `Length`, so my question is whether the `Length` field is needed at all? compared to e.g. a getter func `Length()`?

Reply:
discussed offline, resolving.